raspi/controlserver/controlserver.py

```diff
@@ -22,6 +23,18 @@
 wifi_search_folder = os.path.join(dir_path, '../access_point')
 susiconfig = '/home/pi/SUSI.AI/bin/susi-config'
 
+config_json = json_config.connect('/home/pi/SUSI.AI/config.json')
+
+def get_token():
+    url = 'http://api.susi.ai/aaa/login.json?type=access-token'
+    login = config_json['login_credentials']['email']
+    password = config_json['login_credentials']['password']
+    PARAMS = {
+        'login':login,
+        'password':password,
+    }
```

Comment:
Can we try to use `susi_config` consistently? Something like
```
import susi_config # somehow this should work but probably loading path is not set up
login = susi_config.get_set('susi.user')
password = susi_config.get_set('susi.pass')
```

This way we can change the internal representation of the configuration file without changing any other code.

Probably it is better to move `susi_config` into `pythonmods` as library, and add a trivial wrapper for the main function to the path.

What do you think?

Reply:
Agree.

Reply:
@norbusan I agree too, I'll make the changes, thank you.

Reply:
I have already converted susi-config to a libray `susi_config` in `pythonmods` that can be loaded:
```
import susi_config
cfg = susi_config.SusiConfig(DirectoryOfConfigJson)
user = cfg.set_get('susi.user')
pass = cfg.set_get('susi.pass')
```